### PR TITLE
add v1.3.1, v1.3.3, v1.3.4 of proj4js

### PR DESCRIPTION
**Pull request by maruilian11**
add missing version of [proj4js](https://cdnjs.com/libraries/proj4js)
i didn't find out v1.3.2 tag.

https://github.com/proj4js/proj4js/tree/1.3.1
https://github.com/proj4js/proj4js/tree/v1.3.3
https://github.com/proj4js/proj4js/tree/v1.3.4

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Why don't you minify `amd/proj4.js`?

**Comment by maruilian11** [comment]
@PeterDaveHello i thought `proj4.js` under version folder is already the minified file.

**Comment by PeterDaveHello** [comment]
screenshot from 2016-03-12 21 46 05

You are just GUESSING, it's careless that you did not check but just guess ...

**Comment by maruilian11** [comment]
@PeterDaveHello i will check it next time.

File: ajax/libs/proj4js/1.3.1/proj4.amd.min.js [hunk 1]
```diff
@@ -0,0 +1 @@
+Payload Too Large
```
Comment (PeterDaveHello) [inline ajax/libs/proj4js/1.3.1/proj4.amd.min.js]: Payload Too Large?

**Comment by maruilian11** [comment]
@PeterDaveHello sorry, i will check the minified file next time.

**Pull request merged**